### PR TITLE
chore: inline peerDependencies

### PR DESCRIPTION
**Pull request by SamuelAlev**
Inline `peerDependencies`, `npm` starting version 7 support auto install of `peerDependencies` however other package manager may not, example `pnpm` that we use in [Frontify/guideline-blocks](https://github.com/Frontify/guideline-blocks) requires to manual installation of `peerDependencies`.

The way to fix it is to manually add the dependencies which is not optimal: https://github.com/Frontify/guideline-blocks/blob/main/.pnpmfile.cjs

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -139,11 +140,18 @@
...
         "react-datepicker": "^4.7.0",
         "react-dnd": "^15.1.1",
         "react-dnd-html5-backend": "^15.1.2",
+        "react-is": "^18.1.0",
         "react-popper": "^2.2.5",
+        "slate": "^0.78.0",
+        "slate-history": "^0.66.0",
+        "slate-hyperscript": "^0.77.0",
+        "slate-react": "^0.79.0",
+        "styled-components": "^5.3.5",
```
Comment (SamuelAlev) [inline package.json:154]: Storybook dependency

File: package.json [hunk 2]
```diff
@@ -139,11 +140,18 @@
...
         "react-colorful": "^5.5.1",
         "react-datepicker": "^4.7.0",
         "react-dnd": "^15.1.1",
         "react-dnd-html5-backend": "^15.1.2",
+        "react-is": "^18.1.0",
         "react-popper": "^2.2.5",
+        "slate": "^0.78.0",
+        "slate-history": "^0.66.0",
+        "slate-hyperscript": "^0.77.0",
+        "slate-react": "^0.79.0",
```
Comment (SamuelAlev) [inline package.json:153]: plate dependency

File: package.json [hunk 3]
```diff
@@ -139,11 +140,18 @@
         "@xstate/react": "^1.6.3",
         "date-fns": "^2.28.0",
         "framer-motion": "^6.2.9",
+        "immer": "^9.0.12",
```
Comment (SamuelAlev) [inline package.json:143]: xstate dependency

File: package.json (hunk 2 above)
Comment (SamuelAlev) [inline package.json:153]: plate dependency

**Review by triggertoo — APPROVED** [review]
👍

**Pull request merged**